doc/source/changes/version_0_28.rst.inc

```diff
@@ -10,6 +10,67 @@
 Miscellaneous improvements
 --------------------------
 
+* functions `local_arrays`, `global_arrays` and `arrays` returns a session excluding arrays starting by `_`
+  by default. To include them, set the flag `include_private` to True (closes :issue:`513`):
+
+    >>> global_arr1 = ndtest((2, 2))
+    >>> _global_arr2 = ndtest((3, 3))
+    >>> def foo():
+    ...     local_arr1 = ndtest(2)
+    ...     _local_arr2 = ndtest(3)
+    ...
+    ...     # exclude arrays starting with '_' by default
+    ...     s = arrays()
+    ...     print(s.names)
+    ...
+    ...     # use flag 'include_private' to include arrays starting with '_'
+    ...     s = arrays(include_private=True)
+    ...     print(s.names)
+    >>> foo()
+    ['global_arr1', 'local_arr1']
+    ['_global_arr2', '_local_arr2', 'global_arr1', 'local_arr1']
+
+* implemented sessions binary operations with non sessions objects (closes :issue:`514` and :issue:`515`):
+
+    >>> s = Session({'arr1': ndtest((2, 2)), 'arr2': ndtest((3, 3))})
+    >>> s.arr1
+    a\b  b0  b1
+     a0   0   1
+     a1   2   3
+    >>> s.arr2
+    a\b  b0  b1  b2
+     a0   0   1   2
+     a1   3   4   5
+     a2   6   7   8
+
+    Add a scalar to all arrays
+
+    >>> # equivalent to s2 = 3 + s
+    >>> s2 = s + 3
+    >>> s2.arr1
+    a\b  b0  b1
+     a0   3   4
+     a1   5   6
+    >>> s2.arr2
+    a\b  b0  b1  b2
+     a0   3   4   5
+     a1   6   7   8
+     a2   9  10  11
+
+    Define what to add (array or scalar) using a dictionary (or a session)
```

Comment:
this example is nifty but likely to confuse users. I would scrap it. If you want another example, show something that is more likely to be used: eg. percentage of relative difference (relative diff * 100), or revalorisation of amounts by time (this could be very useful for some of our users).

Reply:
"revalorisation of amounts by time" --> no idea what you mean... sorry